setup.py

```diff
@@ -8,6 +8,7 @@
     'django-cms>=3.7',
     'django-filer>=1.7',
     'djangocms-attributes-field>=1',
+    'django-treebeard>=4.3,<4.5',
```

Comment:
that should be added in the cms and not here?

Reply:
@FinalAngel Do we have a release of django-cms that have treebeard pinned? If not, then what other option do we have  here, since the  latest and buggy treebeard will be installed.

Reply:
> that should be added in the cms and not here?

This is OK. It's needed for cms 3.8 testing since it's requirements allow installation of treebeard 4.5 where there is a bug.